README.md

```diff
@@ -7,41 +7,122 @@ A CNAB bundle for installing [riff](https://projectriff.io/)
 To install this cnab bundle, you can use any available cnab runtime. The following are installation instructions using [duffle](https://duffle.sh/).
 
 ### Steps
-1. Download the latest duffle release for your operating system from duffle's [release page](https://github.com/deislabs/duffle/releases),
- put it somewhere on your path, and make it executable using, for example, `chmod +x /usr/local/bin/duffle`.
-1. Start a kubernetes cluster as described in the riff [getting started](https://projectriff.io/docs/getting-started/) documentation,
-but stop once you've created a cluster and, if appropriate, given yourself cluster-admin permissions.
- If using `minikube`, please run the following command before starting `minikube`:
+The high level steps to install riff are:
+1. Start a kubernetes cluster
+1. Install a CNAB runtime. We will use [duffle](https://duffle.sh/).
+1. Create service account in order to grant additional permissions like creating CRDs over the default sercice account.
```

Comment:
This is a low level implementation detail of CNAB.

A high level step would be something more like:

3. download and install the riff CNAB bundle into the cluster